app/templates/components/public/session-item.hbs

```diff
@@ -0,0 +1,45 @@
+{{#ui-accordion}}
+  <div class="title">
+    <div class="ui">
+      <h4 class="ui header">{{session.title}}</h4>
+      <div class="ui row">Talk</div>
+    </div>
+    <br>
+    <div class="ui grid">
+      <div class="column twelve wide">
```

Comment:
Let the speaker image, microlocation name and start time be in one row and the session shortAbstract in a different row. That would fix the sessionAbstract speaker image spacing issue

<img width="1023" alt="screen shot 2017-05-24 at 12 06 41 am" src="https://cloud.githubusercontent.com/assets/2404372/26370303/0149f5fe-4015-11e7-88a4-1bd99d97ddab.png">